src/windows/process.rs

```diff
@@ -538,6 +558,38 @@ pub(crate) fn get_system_computation_time() -> ULARGE_INTEGER {
     }
 }
 
+pub(crate) fn get_disk_usage(p: &mut Process){
+    let r = ProcessDiagnosticInfo::try_get_for_process_id(p.pid as u32).ok();
+    if r.is_some(){
+        let r = r.unwrap();
```

Comment:
Please never use `unwrap`. If needed, create an inner function expecting a result and then chain `?` calls.

Reply:
sure, I can do that. Thought if I converted it to an option and then check if it was `Some` that `unwrap` won't cause a panic, perhaps I am wrong about that.

Reply:
In that case, you can use `if let`.